config/environment.d.ts

```diff
@@ -78,6 +78,9 @@ declare const config: {
         authenticator: string;
         keenProjectId?: string;
         analyticsDismissAdblockCookie: string;
+        cookies: {
+            cookieConsent: string,
+        },
```

Comment:
Drive-by: There are some more cookies on L76-77 that should be wrapped up into this if you're going to have it. The names should be normalized as well.

Reply:
Yeah, I got those in [another PR](https://github.com/CenterForOpenScience/ember-osf-web/pull/331/files#diff-695693c5dabaffb064bde5b060195ff7R154), so I figured I'd follow that pattern here and make the conflicts easy to deal with.

The others don't start with `cookie`, but since this one is a metacookie, a cookie indicating consent to cookies, it seemed clearer to name it that way.